--- healthchecker/Makefile.am	2013-06-24 16:03:35.000000000 -0700
+++ healthchecker/Makefile.am.new	2013-09-24 09:51:39.041264347 -0700
@@ -48,7 +48,7 @@
 	rm -rf $(TMP_DIR)
 
 install-data-hook:
-	cp -rf $(CHEETAH_LIB) reports $(CLI_PYTHON_LIB) $(pythonlibdir)
+	cp -rf $(CLI_PYTHON_LIB) $(pythonlibdir)
 
 bdist:
 	test -d $(TMP_DIR) || mkdir $(TMP_DIR)
@@ -57,7 +57,7 @@
 	rm -rf $(TMP_DIR)/$(PROJECT_NAME)
 	mkdir -p $(TMP_DIR)/$(PROJECT_NAME)
 	cp $(PROJECT_NAME) *.py  $(TMP_DIR)/$(PROJECT_NAME)
-	cp -rf $(CHEETAH_LIB) reports $(CLI_PYTHON_LIB) LICENSE $(TMP_DIR)/$(PROJECT_NAME)
+	cp -rf $(CLI_PYTHON_LIB) LICENSE $(TMP_DIR)/$(PROJECT_NAME)
 	cp $(TMP_VER) $(TMP_DIR)/$(PROJECT_NAME)/VERSION.txt
 	(cd $(TMP_DIR); tar cf - $(PROJECT_NAME)) | gzip -9 > $(PROJECT_NAME)-`cat $(TMP_VER)`.tar.gz
 	rm -rf $(TMP_DIR)
