--- etc/static_config.in	2013-10-10 08:22:43.824189750 -0700
+++ etc/static_config.in.new	2013-10-10 08:23:32.983571733 -0700
@@ -1,13 +1,13 @@
-{error_logger_mf_dir, "@DATA_PREFIX@/var/lib/couchbase/logs"}.
+{error_logger_mf_dir, "/var/log/couchbase"}.
 {error_logger_mf_maxbytes, 10485760}.
 {error_logger_mf_maxfiles, 20}.
-{path_config_bindir, "@BIN_PREFIX@/bin"}.
-{path_config_etcdir, "@BIN_PREFIX@/etc/couchbase"}.
-{path_config_libdir, "@BIN_PREFIX@/lib"}.
-{path_config_datadir, "@DATA_PREFIX@/var/lib/couchbase"}.
-{path_config_tmpdir, "@DATA_PREFIX@/var/lib/couchbase/tmp"}.
+{path_config_bindir, "/usr/bin"}.
+{path_config_etcdir, "/etc/couchbase"}.
+{path_config_libdir, "/usr/lib"}.
+{path_config_datadir, "/var/lib/couchbase"}.
+{path_config_tmpdir, "/var/tmp/couchbase"}.
 
-{nodefile, "@DATA_PREFIX@/var/lib/couchbase/couchbase-server.node"}.
+{nodefile, "/var/lib/couchbase/couchbase-server.node"}.
 
 {loglevel_default, debug}.
 {loglevel_couchdb, info}.
