--- Makefile	2013-10-10 04:39:26.612615508 -0700
+++ Makefile.new	2013-10-10 04:41:10.671307313 -0700
@@ -83,9 +83,9 @@
 
 NS_SERVER_LIBDIR := $(DESTDIR)$(PREFIX)/lib/ns_server/erlang/lib/ns_server
 
-PREFIX_FOR_CONFIG ?= $(PREFIX)
+PREFIX_FOR_CONFIG ?= ""
 
-COUCHBASE_DB_DIR ?= $(PREFIX)/var/lib/couchbase/data
+COUCHBASE_DB_DIR ?= /var/lib/couchbase/data
 
 ERLWSH_LIBDIR := $(DESTDIR)$(PREFIX)/lib/ns_server/erlang/lib/erlwsh
 GEN_SMTP_LIBDIR := $(DESTDIR)$(PREFIX)/lib/ns_server/erlang/lib/gen_smtp
@@ -113,24 +113,24 @@
 	mkdir -p $(MLOCKALL_LIBDIR)
 	cp -r deps/mlockall/ebin $(MLOCKALL_LIBDIR)/
 	[ ! -d deps/mlockall/priv ] || cp -r deps/mlockall/priv $(MLOCKALL_LIBDIR)/
-	mkdir -p $(DESTDIR)$(PREFIX)/etc/couchbase
+	mkdir -p $(DESTDIR)/etc/couchbase
 	sed -e 's|@DATA_PREFIX@|$(PREFIX_FOR_CONFIG)|g' -e 's|@BIN_PREFIX@|$(PREFIX_FOR_CONFIG)|g' \
-		 <etc/static_config.in >$(DESTDIR)$(PREFIX)/etc/couchbase/static_config
-	touch $(DESTDIR)$(PREFIX)/etc/couchbase/config
+		 <etc/static_config.in >$(DESTDIR)/etc/couchbase/static_config
+	touch $(DESTDIR)/etc/couchbase/config
 	mkdir -p $(DESTDIR)$(PREFIX)/bin/
 	sed -e 's|@PREFIX@|$(PREFIX)|g' <couchbase-server.sh.in >$(DESTDIR)$(PREFIX)/bin/couchbase-server
 	cp cbbrowse_logs $(DESTDIR)$(PREFIX)/bin/cbbrowse_logs
 	cp cbcollect_info $(DESTDIR)$(PREFIX)/bin/cbcollect_info
 	chmod +x $(DESTDIR)$(PREFIX)/bin/couchbase-server $(DESTDIR)$(PREFIX)/bin/cbbrowse_logs $(DESTDIR)$(PREFIX)/bin/cbcollect_info
-	mkdir -p -m 0770 $(DESTDIR)$(PREFIX)/var/lib/couchbase
-	mkdir -p -m 0770 $(DESTDIR)$(PREFIX)/var/lib/couchbase/logs
+	mkdir -p -m 0770 $(DESTDIR)/var/lib/couchbase
+	mkdir -p -m 0770 $(DESTDIR)/var/lib/couchbase/logs
 	cp ebucketmigrator $(DESTDIR)$(PREFIX)/bin/ebucketmigrator
 	chmod +x $(DESTDIR)$(PREFIX)/bin/ebucketmigrator
 	cp scripts/cbdump-config $(DESTDIR)$(PREFIX)/bin/
 	cp scripts/dump-guts $(DESTDIR)$(PREFIX)/bin/
-	mkdir -p $(DESTDIR)$(PREFIX)/etc/couchdb/default.d
-	sed -e 's|@COUCHBASE_DB_DIR@|$(COUCHBASE_DB_DIR)|g' <etc/capi.ini.in >$(DESTDIR)$(PREFIX)/etc/couchdb/default.d/capi.ini
-	cp etc/geocouch.ini.in $(DESTDIR)$(PREFIX)/etc/couchdb/default.d/geocouch.ini
+	mkdir -p $(DESTDIR)/etc/couchdb/default.d
+	sed -e 's|@COUCHBASE_DB_DIR@|$(COUCHBASE_DB_DIR)|g' <etc/capi.ini.in >$(DESTDIR)/etc/couchdb/default.d/capi.ini
+	cp etc/geocouch.ini.in $(DESTDIR)/etc/couchdb/default.d/geocouch.ini
 
 clean clean_all:
 	@(cd deps/gen_smtp && $(MAKE) clean)
